tardis/scripts/cmfgen2tardis.py

```diff
@@ -4,10 +4,10 @@
 import numpy as np
 import pandas as pd
 
-from tardis.atomic import AtomData
+from tardis.io.atom_data import AtomData
 
 
-atomic_dataset = AtomData.from_hdf5()
+#atomic_dataset = AtomData.from_hdf()
```

Comment:
Why is this just commented?

Reply:
It doesn't work, I left it as a comment for the same reason as the other one that I just replied to.

Reply:
This script is broken into too many levels. Why modifying it right now?

Reply:
@epassaro it created a warning

Reply:
I added a comment above the line to explain why it is broken. If we don't comment that part out, it messes up the API as well as defeating the purpose of getting rid of all warnings -- to make it easy to make sure future pull requests don't mess up the docs